### DISCUSSION TITLE
project session is not launched yet

### DISCUSSION THREAD

**Discussion by Aryaajain119** [github/india#460] [Students]
on 10th oct it was showing the project session will open on 20th oct and today it is showing it will open on 10th oct.
![FRT Portal](https://user-images.githubusercontent.com/91967481/138027023-fa810500-7a77-4015-9eb3-74bad760a721.png)

**Comment by Manish814142** [comment]
facing same issue, even i mailed about it to FRT support.

**Comment by trishna-123** [comment]
Yeah having the same problem.

**Reply by Madhura0205** [reply]
Me as well facing the same issue
